### PR TITLE
updates ready for port

### PR DESCRIPTION
**Pull request by pixie79**

### CODE REVIEW

File: schemas/reference/V82__port.sql [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+-- Column comments
+COMMENT ON COLUMN port.id IS '{"label": "Identifier", "description": "Unique identifying column.", "summaryview": "false"}';
+COMMENT ON COLUMN port.countryid IS '{"label": "Linked country id", "description": "Country link to Country dataset.", "summaryview": "false"}';
+COMMENT ON COLUMN port.commandid IS '{"label": "Linked command id", "description": "Border force command link to command dataset.", "summaryview": "false"}';
+COMMENT ON COLUMN port.locationid IS '{"label": "Linked location id", "description": "Border force location link to location dataset.", "summaryview": "false"}';
+COMMENT ON COLUMN port.name IS '{"label": "Name", "description": "Site name", "summaryview": "true"}';
+COMMENT ON COLUMN port.addressid IS '{"label": "Linked address id", "description": "Links to non Home Office addresses.", "summaryview": "true"}';
+COMMENT ON COLUMN unlocode.subdivision IS '{"label": "County Unlocode", "description": "County from unlocode list", "summaryview": "false", "aliases": "region"}';
```
Comment (JimmySunDance) [inline schemas/reference/V82__port.sql]: Small error in Comment command. 
"unlocode.subdivision"
should be
"port.subdivision"

File: schemas/reference/V82__port.sql [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+COMMENT ON COLUMN port.unlocode IS '{"label": "UN location code", "description": "UN location code", "summaryview": "true"}';
+COMMENT ON COLUMN port.helipad IS '{"label": "Helipad", "description": "Item valid from date.", "summaryview" : "false"}';
+COMMENT ON COLUMN port.fixedpcp IS '{"label": "Fixed PCP", "description": "Fixed PCP location.", "summaryview" : "false"}';
+COMMENT ON COLUMN port.portablepcp IS '{"label": "Portable PCP", "description": "Portable PCP location.", "summaryview" : "false"}';
+COMMENT ON COLUMN port.egates IS '{"label": "Egates", "description": "Egate location.", "summaryview" : "false"}';
+COMMENT ON COLUMN port.islargeport IS '{"label": "Large Port", "description": "Is a large port.", "summaryview" : "false"}';
+COMMENT ON COLUMN port.portsizeid IS '{"label": "BF port size", "description": "Links to Border Force port size list.", "summaryview" : "false"}';
+COMMENT ON COLUMN port.validfrom IS '{"label": "Valid from date", "description": "Item valid from date.", "summaryview" : "false"}';
+COMMENT ON COLUMN port.validto IS '{"label": "Valid to date", "description": "Item valid to date.", "summaryview" : "false"}';
+
```
Comment (JimmySunDance) [inline schemas/reference/V82__port.sql]: missing comment command for "localcode"

File: schemas/reference/V82__port.sql [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+COMMENT ON COLUMN port.geolat IS '{"label": "Latitude", "description": "Geographic latitude", "summaryview": "false"}';
+COMMENT ON COLUMN port.geolong IS '{"label": "Longitude", "description": "Geographic longitude", "summaryview": "false"}';
+COMMENT ON COLUMN port.air IS '{"label": "Air", "description": "Is an Air Port with Runway or Helipad.", "summaryview": "true"}';
+COMMENT ON COLUMN port.land IS '{"label": "Land", "description": "Is a land border crossing.", "summaryview": "ture"}';
+COMMENT ON COLUMN port.sea IS '{"label": "Sea", "description": "Maritime port of entry.", "summaryview": "true"}';
+COMMENT ON COLUMN port.rail IS '{"label": "Rail", "description": "Rail  port of entry.", "summaryview": "true"}';
+COMMENT ON COLUMN port.iata IS '{"label": "IATA", "description": "IATA code", "summaryview": "true"}';
+COMMENT ON COLUMN port.icao IS '{"label": "ICAO code", "description": "ICAO code", "summaryview": "true"}';
+COMMENT ON COLUMN port.unlocode IS '{"label": "UN location code", "description": "UN location code", "summaryview": "true"}';
+COMMENT ON COLUMN port.helipad IS '{"label": "Helipad", "description": "Item valid from date.", "summaryview" : "false"}';
```
Comment (JimmySunDance) [inline schemas/reference/V82__port.sql]: Possibly wrong description here.

File: schemas/reference/V82__port.sql (hunk 2 above)
Comment (pixie79) [reply schemas/reference/V82__port.sql]: fixed

File: schemas/reference/V82__port.sql (hunk 1 above)
Comment (pixie79) [reply schemas/reference/V82__port.sql]: fixed

**Pull request merged**